test/conformance/platform/urPlatformCreateWithNativeHandle.cpp

```diff
@@ -83,7 +83,10 @@ TEST_F(urPlatformCreateWithNativeHandleTest, SuccessWithUnOwnedNativeHandle) {
 TEST_F(urPlatformCreateWithNativeHandleTest, InvalidNullPointerPlatform) {
     for (auto platform : platforms) {
         ur_native_handle_t native_handle = 0;
-        ASSERT_SUCCESS(urPlatformGetNativeHandle(platform, &native_handle));
+        {
+            UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(
+                urPlatformGetNativeHandle(platform, &native_handle));
+        }
```

Comment:
What's the point of the additional scope?

Reply:
Not entirely sure, but the rest of the tests have it.

I think it's because the macro is defined like this:
```c++
#define UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(ret)                                 \
    auto status = ret;                                                         \
    if (status == UR_RESULT_ERROR_UNSUPPORTED_FEATURE) {                       \
        GTEST_SKIP();                                                          \
    } else {                                                                   \
        ASSERT_EQ(status, UR_RESULT_SUCCESS);                                  \
    }
```

So I it's to prevent `status` from leaking out of the macro?

... Although that's probably only an issue if there are multiple `UUR_ASSERT_SUCCESS_OR_UNSUPPORTED` calls in the function. I'll just de-scope it.